### ISSUE TITLE
Installation not working - inconsistent with latest termux?

### ISSUE THREAD

**Issue by clemmrich** [EXALAB/AnLinux-App#417]
I tried to install a version of Ubuntu, but the scripts provided by the app seem to be incompatible with termux. 
- first a minor bug: The app shows a command to install wget, but this is not copied, so the naive installation fails because no wget installed.
- after manually installing wget, the copied command did download a shell script, however, this is not working either:
- various "no such file or directory" messages, no file structure is created, the vncserver-start command does not exist ....
- when looking at the downloaded script, the first errors already occur at   the 4th line of the script: it contains apt-get install xfce4 xfce4-terminal tightvncserver dbus-x11 -y, however tightvncserver and dbus-x11 do not exist in the repositories
- same with next line: apt-get install xfe -y
- the line  wget https://raw.githubusercontent.com/EXALAB/AnLinux-Resources/master/Scripts/DesktopEnvironment/Apt/Xfce4/vncserver-start --no-check-certificate -P /usr/local/bin/ does not create any directory, instead giving error "no such file or directory" 
...

Are there any preparationary steps needed, which are not documented, e.g. installing additional packages or repositories?

**Comment by paolorafael** [comment]
Can confirm, I'm experiencing the same errors as you do on an amlogic s905x4 and a Huawei matepad 11.

**Comment by Ultra980** [comment]
I think you might need to add the x11 repo for those packages

**Comment by davidpiano** [comment]
Not true. The issue is with packages that literally don't exist because they were intentionally removed from the repo. The AnLinux install script doesn't check whether downloads have been successful - if downloads fail, the installer just keeps going, pretending to work on files that don't exist.

**Comment by exalab999** [comment]
![Screenshot_2023-11-18-09-09-07-988_com.termux.jpg](https://github.com/EXALAB/AnLinux-App/assets/33449553/048f36d7-ef07-4b2d-9750-5b4608fd251f)

![Screenshot_2023-11-18-09-10-01-442_com.termux.jpg](https://github.com/EXALAB/AnLinux-App/assets/33449553/25beaa95-478c-4398-ac06-8c333b870c14)

Just tested out with the latest Termux fresh install and everything is working flawlessly.

If its not working for you, probably try reinstall Termux, or check if your internet provider or firewall block connection to the Termux mirrors.
